cabby/geo/map_processing/map_structure.py

```diff
@@ -102,15 +102,18 @@ def get_poi(self) -> Tuple[GeoSeries, GeoSeries]:
       (1) The POI that are not roads; and (2) the roads POI.
     '''
 
-    tags = {'name': True, 'building': True, 'amenity': True}
+    tags = {'name': True,
+        'building': True,
```

Comment:
nit: move first dict element down a line, or indent other elements to first element

Reply:
fixed